src/controllers/adminController.js

```diff
@@ -3,6 +3,17 @@ const Album = require('../models/Album');
 const adminHomePage = async (req, res) => {
   const allAlbums = await Album.find({}).sort({ price: 1 }); // Fetch all albums from the database and sort them by price in ascending order
   res.render('adminPage', { albums: allAlbums });
+  /*
+  #swagger.tags = ['Admin']
+
+  #swagger.summary = 'Admin Home Page'
+  #swagger.description = 'Fetch all albums and render the admin home page.'
+
+  #swagger.responses[200] = {
+    description: 'Admin home page fetched successfully.',
+    schema: { $ref: '#/definitions/Album' }
+  }
+*/
 };
 
 const { updatePriceOfAlbums } = require('../controllers/albumController');
```

Comment:
I'm not sure if there's an easy way to tell swagger-autogen about this endpoint since we're just re-exporting the endpoint from another controller. 

You could try making this it's own controller handler that just calls into the albumController one?

```javascript
const { updatePriceOfAlbums: originalUpdatePrice } = require('../controllers/albumController');
const updatePriceOfAlbums = async (req, res) => {
  /* #swagger.tags = ['Admin'] */
  //etc any other swagger infor
  await originalUpdatePrice(req, res);
}
```

Reply:
Thank you very much. I think its work. But now I cant update price and most likely there is another mistakes, it is very difficult for me to get documentation, that's why I took the admin page😅
<img width="911" alt="swagger" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/6fae523d-7d16-4547-8ce8-60e25cb4dde2">